dashboard/app/models/script.rb

```diff
@@ -876,7 +877,8 @@ def summarize(include_stages=true)
       excludeCsfColumnInLegend: exclude_csf_column_in_legend?,
       teacher_resources: teacher_resources,
       stage_extras_available: stage_extras_available,
-      has_verified_resources: has_verified_resources?
+      has_verified_resources: has_verified_resources?,
+      script_announcements: script_announcements,
```

Comment:
I forget, does ruby complain about trailing comma?

Reply:
Nope. This passed linter :)